index.js

```diff
@@ -161,7 +161,7 @@ function CompassCompiler(inputTree, files, options) {
   this.options = merge(true, this.defaultOptions);
   merge(this.options, options);
   options = this.options;
-  options.files = (options.files instanceof Array) ? options.files : [];
+  options.files = (options.files && options.files.length) ? [].concat.apply([], [options.files]) : [];
```

Comment:
This can lead to have an array of array !

Change it to: `options.files = [].concat(options.files);`